### PR TITLE
TRE-709 Updated FAQ

### PR DESCRIPTION
**Pull request by ameliemaucade**

### CODE REVIEW

File: vipps-recurring-api-faq.md [hunk 1]
```diff
@@ -40,9 +40,9 @@ Document version: 1.6.1.
 * [How can I change partners for a merchant?](#how-can-i-change-partners-for-a-merchant)
 * [At what time during the day are charges made?](#at-what-time-during-the-day-are-charges-made)
 * [How do I check my customer's status?](#how-do-i-check-my-customers-status)
-* [A customer's charge failed, but I did not receive any warning](#a-customers-charge-failed-but-i-did-not-receive-any-warning)
+* [A customer's charge failed, but I did not receive any warning](#a-customers-charge-failed-but-did-not-receive-any-warning)
 * [I don't want a charge to fail the first time the transaction fails (insufficient funds / networking issues etc.)](#i-dont-want-a-charge-to-fail-the-first-time-the-transaction-fails-insufficient-funds--networking-issues-etc)
-* [Can the charge for an Agreement be changed?](#can-the-charge-for-an-agreement-be-changed)
+* [Can the charge for an Agreement be changed?](#can-the-charge-be-different-from-the-agreements-price)
```
Comment (cloveras) [inline vipps-recurring-api-faq.md:45]: Shoyldn't the ToC entry and the actual paragraph heading be the same?
```suggestion
* [Can the charge for an Agreement be different from the agreement price?](#can-the-charge-be-different-from-the-agreements-price)
```

File: vipps-recurring-api-faq.md [hunk 2]
```diff
@@ -51,10 +51,11 @@ Document version: 1.6.1.
 * [If a user's card expires: What happens on the next charge?](#if-a-users-card-expires-what-happens-on-the-next-charge)
 * [What happens to pending charges if the user deletes the payment card?](#what-happens-to-pending-charges-if-the-user-deletes-the-payment-card)
 * [How does a user see any charges I send?](#how-does-a-user-see-any-charges-i-send)
-* [If a user changes the default payment card in Vipps, can new charges be made to that card?](#if-a-user-changes-the-default-payment-card-in-vipps-can-new-charges-be-made-to-that-card)
+* [If a user changes the default payment card in Vipps, can new charges be made to that card?](#if-a-user-adds-or-updates-a-card-in-vipps-will-new-recurring-charges-be-made-to-that-card)
```
Comment (cloveras) [inline vipps-recurring-api-faq.md:54]: Same here.

**Review by cloveras — CHANGES_REQUESTED** [review]
Nice to get the push info in here! I have some minor comments about wording.

File: vipps-recurring-api-faq.md (hunk 1 above)
Comment (ameliemaucade) [reply vipps-recurring-api-faq.md:45]: Yes they should, forgot to update them. Thanks !

**Review by rognlien — APPROVED** [review]

**Pull request merged**